libraries/lib-wave-track/WaveTrack.h

```diff
@@ -1249,7 +1278,7 @@ class WAVE_TRACK_API WaveTrack final
     @param backup whether the duplication is for backup purposes while opening
     a project, instead of other editing operations
     */
-   bool InsertClip(WaveClipHolder clip, bool backup = false);
+   bool InsertClip(WaveClipHolder clip, bool newClip, bool backup = false);
```

Comment:
IIUC, this is to know whether the clip's colour should be overwritten, and this depends on whether it's the copy of a clip from another track or a freshly created clip.
If I'm right, I'd much prefer it to be called `fromOtherTrack` or similar. It denotes precisely the intention, and I don't really know what "new clip" means anyways.